groot/Source.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+
+int main() {
+
+	int user_input = 0;
+	std::cout << "Please enter a number: ";
+	std::cin >> user_input;
```

Comment:
Can this function fail? What happens if I insert 'Hello world' and press 'Enter'?